cmake/projects/getopt/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+if(MSVC) #only for Visual Studios, all other posix compiliers should have getopt
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    getopt
+    VERSION
+    "1.0.0-p0"
+    URL
+    "https://github.com/caseymcc/getopt/archive/1.0.0-p0.tar.gz"
```

Comment:
You can use hunter-packages now:
* https://github.com/hunter-packages/wingetopt